tests/ci_visibility/api/fake_runner_all_fail.py

```diff
@@ -0,0 +1,129 @@
+"""Fake test runner where all tests pass"""
+
+from multiprocessing import freeze_support
+import sys
+
+from ddtrace.ext.ci_visibility import api
+from ddtrace.internal.ci_visibility.utils import take_over_logger_stream_handler
+
+
+def _make_excinfo():
+    try:
+        raise ValueError("This is a fake exception")
+    except ValueError:
+        return api.CIExcInfo(*sys.exc_info())
+
+
+def main():
+    take_over_logger_stream_handler()
+
+    api.enable_ci_visibility()
+
+    # START DISCOVERY
```

Comment:
This might be a good use case for a function

Reply:
Yeah. I kinda wanted to leave myself flexibility to change the tests individually. Tests is the one place where I don't mind not being DRY.